src/main/kotlin/org/rust/ide/presentation/RsPsiRenderer.kt

```diff
@@ -130,9 +130,10 @@ open class RsPsiRenderer(
         if (typeParameterList != null && renderGenericsAndWhere) {
             appendTypeParameterList(sb, typeParameterList)
         }
-        val whereClause = ta.whereClause
-        if (whereClause != null && renderGenericsAndWhere) {
-            appendWhereClause(sb, whereClause)
+        if (renderGenericsAndWhere) {
+            for (whereClause in ta.whereClauseList) {
+                appendWhereClause(sb, whereClause)
+            }
```

Comment:
This code will append both where clauses in the same place (before a type) like `type A<T> where T: Foo where T: Bar `. You can write a test for this case in `ImplementMembersHandlerTest`. But I don't think this bug is really important since GAT defaults are unstable and nobody would write 2 where clauses at the same time.